client/components/main/editor.js

```diff
@@ -153,54 +153,33 @@ BlazeComponent.extendComponent({
                   });
                 }
               },
-
               onImageUpload(files) {
                 const $summernote = getSummernote(this);
                 if (files && files.length > 0) {
                   const image = files[0];
-                  const currentCard = Utils.getCurrentCard();
+                  const currentCard = Cards.findOne(Session.get('currentCard'));
```

Comment:
@imajus
This was necessary for mobile devices because the card detail is opened as popup.
Is there something what doesn't work so you changed it?